app.js

```diff
@@ -12,12 +12,9 @@ var regExp = /[^a-z]+/gi;
 //  Do this when the JSON loads
 request.onload = function() {
     // Begin accessing JSON data here
-    var data = JSON.parse(this.response);
+    var data = JSON.parse(request.response);
```

Comment:
Nice Work!! Love that you made an API request. This is ```async``` programming. We'll be discussing this more...